hack/jenkins/job-configs/kubernetes-jenkins/kubernetes-update-jenkins-jobs.yaml

```diff
@@ -4,7 +4,10 @@
     triggers:
         - timed: 'H/15 * * * *'
     builders:
-        - shell: 'curl -fsS https://raw.githubusercontent.com/kubernetes/kubernetes/master/hack/jenkins/update-jobs.sh | /bin/bash -'
+        - shell: |
+            curl -fsS https://raw.githubusercontent.com/kubernetes/kubernetes/master/hack/jenkins/update-jobs.sh > update-jobs.sh
+            chmod +x update-jobs.sh
+            ./update-jobs.sh "hack/jenkins/job-configs:hack/jenkins/job-configs/kubernetes-jenkins"
```

Comment:
JJB lets you join directories as in your `PATH` variable. I just put global stuff in `job-configs``, the rest is in one of the other two folders depending on which Jenkins instance to run it in.